packages/graphing/src/labels.jsx

```diff
@@ -51,11 +51,12 @@ class RawLabel extends React.Component {
 
   render() {
     const { text, side, graphProps, classes } = this.props;
+    const { size, domain, range } = graphProps;
+    const totalHeight = size.height + range.padding * 2;
+    const totalWidth = size.width + domain.padding * 2;
```

Comment:
Is there a chance domain.padding/ range.padding get here as undefined? If yes, then make sure you initialize them with 0/1 (which one works for you).

Reply:
Same for size height and width